rel/files/ejabberd.cfg

```diff
@@ -173,6 +166,19 @@
         ]}
     ]},
 
+  %% MongooseIM HTTP API it's important to start it on localhost
+  %% or some private interface only (not accessible from the outside)
+  %% At least start it on different port which will be hidden behind firewall
+
+  { {{metrics_http_endpoint}} , ejabberd_cowboy, [
```

Comment:
maybe just `http_endpoint`/`rest_endpoint` ? It is not only about metrics endpoint.